src/nav/full-stack-observability.yml

```diff
@@ -723,7 +723,7 @@ pages:
             pages:
               - title: Infrastructure logging behavior
                 path: /docs/infrastructure/infrastructure-troubleshooting/troubleshoot-logs/infrastructure-agent-logging-behavior
-              - title: 'Agent doesn&#039;t start, no logs'
+              - title: 'Agent doesn''t start, no logs'
```

Comment:
Looks like this one added 2 quotes for some reason

Reply:
I don't know why this is happening. I updated the `replace` to use a global regex, but I'm still seeing this odd behavior. Any ideas why that might be happening?

Reply:
🤦 It looks busted in the YAML, but it actually renders in the desired format. Since this solving the issue, I'm going to say this is good enough for now. We can always manually tweak this _one_ spot if it starts to render weird.

![Screen Shot 2021-02-12 at 2 43 49 PM](https://user-images.githubusercontent.com/1946433/107830679-3be38380-6d41-11eb-982e-2c5905626b99.png)

Reply:
Ha, I probably shouldn't have assumed it was incorrect. Good find.